dependency-manager-tests/spm/SPMProject/AppDelegate.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AppDelegate.swift
+//  SPMProject
+//
```

Comment:
Please fix the license header. IDK why `./tools/license/check-license.sh` did pass on CI even though it captured a file with no license:

<img width="778" alt="Screenshot 2020-03-26 at 16 18 04" src="https://user-images.githubusercontent.com/2358722/77663682-a058f100-6f7d-11ea-8fa9-c634568c5ad5.png">

But no failure on CI is a separate issue, I'll put this into JIRA.